### PR TITLE
feat: Add experimental ARM kernel image

### PR DESCRIPTION
**Pull request by Callisto13**
1 of 3

### CODE REVIEW

File: .github/workflows/arm-kernel-images.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      contents: read
+      packages: write
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [ 5.10.77 ]
+    env:
+      KERNEL_VERSIONS: ${{ matrix.version }}
+    steps:
+      - uses: actions/checkout@v2
```
Comment (yitsushi) [inline .github/workflows/arm-kernel-images.yml:44]: note: there is a `checkout@v3`, some of my pet project jobs started to throw notices about it. In these cases it's only a `v2` to `v3` replace.

(yes copy paste)

**Review by yitsushi — APPROVED** [review]

File: .github/workflows/arm-kernel-images.yml (hunk 1 above)
Comment (yitsushi) [reply .github/workflows/arm-kernel-images.yml:44]: https://github.com/weaveworks-liquidmetal/image-builder/issues/46

Comment (Callisto13) [reply .github/workflows/arm-kernel-images.yml:44]: uuugh but then i have to dismiss all your reviews 😂

Comment (yitsushi) [reply .github/workflows/arm-kernel-images.yml:44]: that's why I created a new issue for that ;)

Comment (Callisto13) [reply .github/workflows/arm-kernel-images.yml:44]: noice ;)

**Pull request merged**